runtime/development/src/weights/pallet_permissions.rs

```diff
@@ -31,33 +31,33 @@ use sp_std::marker::PhantomData;
 /// Weights for pallet_permissions using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-	fn add_as_admin() -> Weight {
-		(70_000_000 as Weight)
+	fn add_permission_admin() -> Weight {
+		(31_499_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
-	fn add_as_editor() -> Weight {
-		(55_000_000 as Weight)
+	fn add_permission_editor() -> Weight {
```

Comment:
The name of these weights has been updated, the `_permission_` bit was dropped due to redundancy.